src/measure_theory/measure/hausdorff.lean

```diff
@@ -952,3 +935,215 @@ e.isometry.hausdorff_measure_image (or.inr e.surjective) s
 by rw [← e.image_symm, e.symm.hausdorff_measure_image]
 
 end isometry_equiv
+
+open_locale pointwise
+
+lemma hausdorff_measure_smul
+  {E : Type*} [normed_add_comm_group E] [inner_product_space ℝ E]
+  [measurable_space E] [borel_space E]
+  (d : ℝ) (r : ℝ) (hr : r ≠ 0) (hd : 0 ≤ d) (s : set E) :
```

Comment:
It's not immediately obvious to me whether `0 ≤ d` is needed; I just added it because it made the proof easier.